common/common.scss

```diff
@@ -40,7 +40,7 @@
   }
 
   .tiles-grid-item:hover {
-    background-color: darken($secondary, 2%);
+    background-color: var(--secondary-hover);
     lighten: ($primary, 70%);
```

Comment:
I believe this is intended to lighten the foreground, ie main written colour, so that it makes it even more vivid and full of contrast.  You should keep the darken here because it's supposed to darken when you hover.